sklearn/preprocessing/label.py

```diff
@@ -661,6 +668,20 @@ class MultiLabelBinarizer(BaseEstimator, TransformerMixin):
     >>> list(mlb.classes_)
     ['comedy', 'sci-fi', 'thriller']
 
+    Perform a one-hot encoding for y labels
```

Comment:
Oh right. But internal to scikit-learn, the flat vector version is a more efficient representation of binary labels...